listenbrainz/influx-writer/influx-writer.py

```diff
@@ -55,7 +55,7 @@ def connect_to_rabbitmq(self):
                 self.connection = pika.BlockingConnection(pika.ConnectionParameters(host=config.RABBITMQ_HOST, port=config.RABBITMQ_PORT))
                 break
             except Exception as e:
-                self.log.error("Cannot connect to rabbitmq: %s, retrying in 2 seconds")
+                self.log.error("Cannot connect to rabbitmq: %s, retrying in 2 seconds", str(e))
```

Comment:
FYI, the logging framework lets you pass format arguments as parameters. The idea here is that it will only perform the interpolation if this debugging level is turned on. If you use `%` then you have to do the interpolation each time you call the log function, even if the message isn't shown - this could be expensive if you log lots of times in a loop.